others/connect_four_game.cpp

```diff
@@ -0,0 +1,205 @@
+/**
+* @file
+* @brief [Connect Four](https://en.wikipedia.org/wiki/Connect_Four)
+*
+* @details
+* This is a user interactive Connect-Four game for two players.
+* A board with 6 rows and 8 columns is displayed.
+* Two players are assigned the symbols "x" and "o".
+* They take turns to choose the column where their respective symbol can be inserted.
+* A player wins when 4 spaces are connected continuously: Horizontally, Vertically, Diagonally Right, or Diagonally Left.
+* The program declares the winner.
+*
+* @author [Arunabh Bhattacharya](https://github.com/arunabh-cloud)
+*/
+#include <iostream>   //for input/output
+#include <array>    //for game board
+
+using std::cout;
+using std::cin;
+
+// Global constants COLS and ROWS for the standard (row * column) 2D game board
+const int COLS = 8;
+const int ROWS = 6;
+
+// Function prototypes
```

Comment:
Prototypes are not needed here; they just create more lines of code. It'd be nice if you could remove them. :)